corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    l = name.capitalize
+    l[0, 10]
```

Comment:
- [ ] You don't have to create a new constant l, you already have name => name.capitalize and name[0, 10]

Reply:
That's the same! so why didn't you approve it ?